cli/ts/index.ts

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
why removing this? does it work without?

Reply:
Right, forgot to return it back.